src/frontend/src/lib/components/tokens/TokenLogo.svelte

```diff
@@ -1,11 +1,13 @@
 <script lang="ts">
+	import { nonNullish } from '@dfinity/utils';
 	import Logo from '$lib/components/ui/Logo.svelte';
 	import { i18n } from '$lib/stores/i18n.store';
 	import type { Token } from '$lib/types/token';
 	import { replacePlaceholders } from '$lib/utils/i18n.utils';
 
 	export let token: Token;
 	export let color: 'dust' | 'off-white' | 'white' = 'dust';
+	export let tokenCount: number | undefined = undefined;
```

Comment:
Should we have a single property for both count and network because the first exclude the second. In addition, `showNetworkIcon` is by default to `true` so it's a bit misleading to have it default but hidden if token count is set. WDYT?

Reply:
yes, make sense: I refactored a bit, maybe check last commit, WDYT?